integration_tests/test_end_to_end.py

```diff
@@ -99,7 +98,7 @@ def check_open_with_dc(index):
     assert data_array.shape
     assert (data_array != -999).any()
 
-    data_array = dc.load(product='ls5_nbar_albers', measurements=['blue'], time='1992-03-23T23:14:25.500000')
+    data_array = dc.load(product='ls5_nbar_albers', measurements=['blue'], time='1992-03-23 23:14:25.500000')
```

Comment:
It'd be nice to not have broken this. It's probably not a crucial use case, but flexible date handling is good.